ingest-app/src/cmr/ingest/validation/validation.clj

```diff
@@ -383,12 +386,32 @@
   [context]
   (let [kms-index (kms-fetcher/get-kms-index context)]
     {:MeasurementIdentifiers (measurement-validation
-                              kms-index msg/measurements-not-matches-kms-keywords)}))
+                              kms-index msg/measurements-not-matches-kms-keywords)
+
+     :RelatedURLs (v/every [{:Format (match-kms-keywords-validation-single
+                                      kms-index
+                                      :granule-data-format
+                                      msg/getdata-format-not-matches-kms-keywords)}
+                            {:MimeType (match-kms-keywords-validation-single
+                                        kms-index
+                                        :mime-type
+                                        msg/mime-type-not-matches-kms-keywords)}])}))
+
+(defn- variable-keyword-validations-non-ignorable
```

Comment:
unignorable?

Reply:
I can try that, I have not been happy with this name and welcome suggestions.